src/Database/Connection.php

```diff
@@ -125,23 +125,25 @@ public function __construct(array $config)
         $this->_config = $config;
         [self::ROLE_READ => $this->readDriver, self::ROLE_WRITE => $this->writeDriver] = $this->createDrivers($config);
 
+        /**
         if (!empty($config['log'])) {
             $this->enableQueryLogging((bool)$config['log']);
         }
+         */
```

Comment:
This is broken because query logging is not present inside the connection anymore